src/frontends/tensorflow_lite/include/openvino/frontend/tensorflow_lite/extension/conversion.hpp

```diff
@@ -12,8 +12,10 @@ namespace ov {
 namespace frontend {
 namespace tensorflow_lite {
 
-class TENSORFLOW_LITE_FRONTEND_API ConversionExtension : public ConversionExtensionBase {
+class ConversionExtension : public ConversionExtensionBase {
```

Comment:
We can break build for customers who rely on dynamic linking

Reply:
what is the use case? why they rely on it?